osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -184,6 +195,30 @@ protected override void OnHoverLost(HoverLostEvent e)
                 base.OnHoverLost(e);
             }
 
+            private bool pauseOnFocusLost;
+
+            public bool PauseOnFocusLost
+            {
+                set
+                {
+                    if (pauseOnFocusLost == value)
+                        return;
+
+                    pauseOnFocusLost = value;
+                    updateActive();
+                }
+            }
+
+            private void updateActive()
+            {
+                if (!pauseOnFocusLost) return;
+
+                if (gameActive.Value)
+                    AbortConfirm();
+                else
+                    BeginConfirm();
```

Comment:
This sometimes exits me from gameplay if the game is deactivated right as PlayerLoader finishes loading the Player. Intended?

Reply:
Not intended. `BeginConfirm` was getting triggered twice (once from `PlayerLoader`).